com.photopea.Photopea.yml

```
@@ -0,0 +1,22 @@
+app-id: com.photopea.Photopea
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: photopea
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

unless it doesn't support wayland.